docs/environment.md

```diff
@@ -308,7 +339,7 @@ SET GCM_GITHUB_AUTHMODES="oauth,basic"
 export GCM_GITHUB_AUTHMODES="oauth,basic"
 ```
 
-**Also see: [credential.gitHubAuthModes](configuration.md#credentialgitHubAuthModes)**
+**Also see: [credential.gitHubAuthModes](configuration.md#credential.gitHubAuthModes)**
```

Comment:
GitHub link fragments ignore periods `.` in headings, so this change breaks the link.